test/lit/passes/optimize-instructions-gc.wast

```diff
@@ -1247,7 +1247,6 @@
         (ref.as_non_null
           (local.get $x)
         )
-        (rtt.canon $struct)
```

Comment:
What was this? (Was this a correct syntax?)

Reply:
This was just incorrect syntax I happened to notice. Our parser atm will ignore extra items at the end in most cases (unless it has an explicit check for the length), and this is one such case.